src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -800,7 +906,7 @@ protected override void BeginProcessing()
                     case WaitForServiceTypes.WinRM:
                         break;
                     case WaitForServiceTypes.PowerShell:
-                        _powershell = System.Management.Automation.PowerShell.Create();
+                        _powershell = global::System.Management.Automation.PowerShell.Create();
```

Comment:
Is this change necessary & what does it accomplish here? Never seen this sort of syntax in c# before, and I don't know if it's used anywhere else in the repo here.

Reply:
This forces the call to search in the global namespace, otherwise the 'System' prefix grabs the local System variable. However, this likely won't be an issue after the the reason codes are converted to enums.